pkg/cmd/grafana-cli/utils/command_line.go

```diff
@@ -92,6 +88,25 @@ func (c *ContextCommandLine) PluginRepoURL() string {
 	return c.String("repo")
 }
 
+func (c *ContextCommandLine) Config() (*setting.Cfg, error) {
```

Comment:
Imported from this [PR](https://github.com/grafana/grafana/pull/98971/files#diff-f156716f016cbd19cee75dea06358828f2ab28254162bed132f1d84a304ecccdR58) :bank: